source/init.html.md.erb

```diff
@@ -1358,11 +1358,11 @@ The type of error that is being reported.
 |reference|string|false|none|An opaque string identifying this check to the provider. No two checks from the same provider should have the same reference.|
 |custom_data|[CustomData](#schemacustomdata)|false|none|Use this to store any state required by later requests relating to this check. The intent is for your integration to be completely stateless itself, to make it simpler to deploy and maintain. Note: if present, this *must* be an object, rather than an array or other JSON value. However, the keys and values in this object can be anything you like, although we don't recommend storing large amounts of data here, as it will be passed in on every following request to your integration relating to this check.|
-|provider_data|[ProviderData](#schemaproviderdata)|false|none|The structured JSON data returned by the data provider, or a representation of the provider's response, to allow PassFort to investigate issues that may arise with your integration.|
+|provider_data|[ProviderData](#schemaproviderdata)|false|none|The structured JSON data returned by the data provider, or a representation of the provider's response, to allow Passfort to investigate issues that may arise with your integration.|
 |check_output|[EntityData](#schemaentitydata)|false|none|none|
-|charges|[[Charge](#schemacharge)]|false|none|A list of charges incurred while running this check. Charges may only be returned if no errors occurred (i.e. a result was generated) and the check is being resold (the commercial relationship is with PassFort). For demo checks, charges may be returned, but they will not be deducted from the institution's credit. If charges are returned, the total amount charged must be no greater than the maximum cost-per-check specified in the integration's configuration.|
-|warnings|[Warnings](#schemawarnings)|false|none|Similar to 'errors' except that the sub_type and data fields are not present and providing warnings will not cause PassFort to consider the check as errored.|
-|errors|[Errors](#schemaerrors)|false|none|A list of errors that occurred while running this check. If this is provided and it's not empty, the result of this check will be considered to be errored by PassFort, regardless of the other contents of this response.|
+|charges|[[Charge](#schemacharge)]|false|none|A list of charges incurred while running this check. Charges may only be returned if no errors occurred (i.e. a result was generated) and the check is being resold (the commercial relationship is with Passfort). For demo checks, charges may be returned, but they will not be deducted from the institution's credit. If charges are returned, the total amount charged must be no greater than the maximum cost-per-check specified in the integration's configuration.|
+|warnings|[Warnings](#schemawarnings)|false|none|Similar to 'errors' except that the sub_type and data fields are not present and providing warnings will not cause Passfort to consider the check as errored.|
```

Comment:
Can we link "errors" to the "errors section"? Format <code>sub_type</code> as code? Change "are not" to "aren't"